pkg/controller/account/account_controller.go

```diff
@@ -631,6 +625,81 @@ func (r *ReconcileAccount) statusUpdate(reqLogger logr.Logger, account *awsv1alp
 	return err
 }
 
+func (r *ReconcileAccount) setStateFailed(reqLogger logr.Logger, currentAcctInstance *awsv1alpha1.Account, msg string) error {
+	reqLogger.Info(msg)
+	// Upodate the account status (state and condition) to Failed
+	utils.SetAccountStatus(reqLogger, currentAcctInstance, msg, v1alpha1.AccountFailed, AccountFailed)
+	err := r.statusUpdate(reqLogger, currentAcctInstance)
+	if err != nil {
+		return err
+	}
```

Comment:
Redundant with [L643-5 below](https://github.com/openshift/aws-account-operator/pull/425/files#diff-32f12988d73ded1726ea7c67c8cb1f90R643-R645). Remove one or the other.

Reply:
Done!